vipps-recurring-api.md

```diff
@@ -444,6 +452,65 @@ until the user wishes to resume the subscription. It's also recommended to updat
 the `productDescription` field of the agreement so the user can see that the
 subscription is paused in the Vipps app.
 
+## Userinfo
+
+**Important:** This is an early draft, this should be considered pilot
+functionality that we are currently rolling out in our test environemnt.
+Swagger will be fully updated
+shortly
+
+Vipps offers a functionality to ask for a generic consent to access Userinfo.
+This is based on the
+[Vipps Login](https://github.com/vippsas/vipps-login-api)
+solution, but merchants can seamlessly combine the two functionalites
+in a single user session. Combining both the userinfo and payment elements
+requires the merchant to be registered with both Vipps Login and
+Vipps Recurring APIs.
+
+When you initiate a payment add the parameter `scope` to ask for a user's
```

Comment:
Should we add an example json just to make sure they get how to add the scopes?

Reply:
Yes but I was thinking that could wait until Mats was completely done with the implementation